### PR TITLE
You can smash the bulbs in floodlights

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl: Kor
balance: You can now smash the bulbs out of floodlights rather than having to entirely destroy the object.
/:cl:

### CODE REVIEW

**Comment by TechnoAlchemisto** [comment]
Mr. Phaeron,

It begrieves me to hear that you've been betrayed by Mr. Zhang, a close associate and maybe even friend. I know that things may be looking dark, but at times like these, we have to think of fond memories, and believe that things are going to get better. This reminds me of when you and I were exploring that old warehouse, down by the docks, and you fell through that old shed and fractured your tibia(kneecap).  Instead of losing hope, you looked up at me and told me "Please call an ambulance", I'll never forget the look on your face that day, an unbreaking smile. 

Think of this betrayal by Mr. Zhang, as a broken tibia(kneecap), just keep smiling and you'll be just fine (after a few months of physical therapy LOL). 

P.S - Really sad you didn't make it to my birthday party, but I understand that you're a busy guy! Can't wait to see you at the wedding this winter(Don't get me and Ty anything too fancy, 50 dollars max.)

Can't wait to see you Best Man :D!

**Comment by ExcessiveUseOfCobblestone** [comment]
If I go can I send you help @TechnoAlchemisto

File: code/modules/power/floodlight.dm [hunk 1]
```diff
@@ -104,3 +104,13 @@
 /obj/machinery/power/floodlight/attack_ai(mob/user)
 	attack_hand(user)
 	..()
+
+/obj/machinery/power/floodlight/take_damage(damage_amount, damage_type = BRUTE, damage_flag = 0, sound_effect = 1)
```
Comment (Cyberboss) [inline code/modules/power/floodlight.dm]: use obj_break and integrity failure

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Cyberboss** [comment]
changelog

**Comment by KorPhaeron** [comment]
@Cyberboss should be done

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**